lib/world_cup.rb

```diff
@@ -0,0 +1,24 @@
+require 'pry'
+class WorldCup
+  attr_reader :year,
+              :teams
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+  end
+
+  def active_players_by_position(inquired_position)
```

Comment:
I love the logic here, but I would try to break this into multiple methods to maintain SRP.  You might end up with the following methods: `active_teams` and `active_players_by_position`.